app/models/team.rb

```diff
@@ -0,0 +1,6 @@
+class Team < ApplicationRecord
+  has_many :project_teams
+  has_many :projects, through: :project_teams
+
+  has_many :team_users
```

Comment:
Association for users

Reply:
user model is not created yet... so thats why i did not add it here